src/components/bookmarklet/BookmarkletButtons.vue

```diff
@@ -113,7 +119,7 @@ export default {
       this.go(`/bookmarklet/popup/${this.previousStep}`)
     },
     onClickCancel() {
-      this.$announcer.polite('Canceled.')
+      this.$announcer.polite(`${this.cancelButtonLabel}ed.`)
```

Comment:
@johncrossman If `cancelButtonLabel = 'Close'`, will the announcer say "Closeed"?

Reply:
Good catch! I'll fix.